### DISCUSSION TITLE
Linking to files does not work with my Github Pages site.

### DISCUSSION THREAD

**Discussion by sarahthebest** [community/community#81356] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone!

I need help with a problem I have encountered on my GitHub [page](https://sarahthebest.github.io/). Repo [here ](https://github.com/sarahthebest/sarahthebest.github.io)

All of the get requests respond with 404 error. None of the file links eg. assets or scripts gets loaded. I have checked my paths and they seem to be correct. I have no clue what is causing this problem.

**Comment by sarahthebest** [comment]
Solved it!

**Reply by Sugarkom7** [reply]
How?
